photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -262,7 +264,7 @@ object RandomEffectCoordinate {
             case _ =>
               throw new IllegalStateException("Either a initial random effect model or data should be present!")
           }
-        modelsAndTrackers.persist(StorageLevel.MEMORY_ONLY_SER)
+        modelsAndTrackers.persist(StorageLevel.MEMORY_AND_DISK_SER)
```

Comment:
Why do we need to change the StorageLevel?

Reply:
changed back to the original storage level.